### PR TITLE
Add competition & Tech day reminder email

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Comment by antoinepetty** [comment]
Need to change tech day description to be for volunteers instead of teams

File: SR2020/2020-02-20-competition-reminder.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+We provide a space for you to work in, with power and internet access, as well as volunteers able to help you with your kits and hands-on guidance with your robots.
+
+The next tech days coming up will be on:
+
+| Date | Location | 
+|February 29th |[London (Thread, Whitechapel)](https://studentrobotics.org/events/sr2020/london-tech-day-february/) |
+| March 7th | [University of Southampton](https://studentrobotics.org/events/sr2020/southampton-tech-day-march/) |
+| March 28th | [London (Thread, Whitechapel)](https://studentrobotics.org/events/sr2020/london-tech-day-march/) |
+
+If you’re free to help out on a tech day, then just turn up! The events pages have all the details. Don’t worry if you’re new, we’d be happy to help you out with assisting teams or doing other things for the competition!
```
Comment (RealOrangeOne) [inline SR2020/2020-02-20-competition-reminder.md]: There's actually a signup form in drive for volunteers for tech days.

File: SR2020/2020-02-20-competition-reminder.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+Hello all,
+
+Quick update on the volunteering opportunities:
+## We still need volunteers for the competition!
+
+The dates will be April 18th - 19th, with set-up days on the 17th and 16th if you’re available! The venue is RUSU, the University of Reading Student Union.
+
+Accommodation will be free for volunteers, and we’ll provide lunch on the competition days. We may also reimburse travel depending on the budget, so keep your train ticket receipts!
```
Comment (PeterJCLaw) [inline SR2020/2020-02-20-competition-reminder.md]: Some tweaks:
```suggestion
Accommodation will be provided for volunteers, and we’ll provide lunch on the competition days. We may also reimburse travel depending on the budget, so keep your travel receipts!
```

- "provided" over "free" clarifies that we've also done the organising
- "travel" over "train tickets" accounts for other modes of travel

File: SR2020/2020-02-20-competition-reminder.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+to: All Volunteers
+subject: We need competition volunteers!
+---
+
+Hello all,
+
+Quick update on the volunteering opportunities:
+## We still need volunteers for the competition!
```
Comment (PeterJCLaw) [inline SR2020/2020-02-20-competition-reminder.md]: markdown nit: newlines before headings.
```suggestion

## We still need volunteers for the competition!
```

File: SR2020/2020-02-20-competition-reminder.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+The dates will be April 18th - 19th, with set-up days on the 17th and 16th if you’re available! The venue is RUSU, the University of Reading Student Union.
+
+Accommodation will be free for volunteers, and we’ll provide lunch on the competition days. We may also reimburse travel depending on the budget, so keep your train ticket receipts!
+
+Please sign up if you fancy coming: https://forms.gle/foHy82uY98wiPm587
+
+**The deadline for applying to ensure you get accommodation is March 14th!!**
+
+## Helping with Tech Days
```
Comment (PeterJCLaw) [inline SR2020/2020-02-20-competition-reminder.md]: markdown nit: newlines after headings
```suggestion
## Helping with Tech Days

```

File: SR2020/2020-02-20-competition-reminder.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Tech Days are opportunities for teams to spend a whole day working on their robot with lots of help available from us. They’re also an opportunity to see how other teams are doing or get more direct help with your robots.
```
Comment (PeterJCLaw) [inline SR2020/2020-02-20-competition-reminder.md]:
```suggestion
Tech Days are opportunities for teams to spend a whole day working on their robot with lots of help available from us. They’re also an opportunity to meet and get to know other volunteers.
```

File: SR2020/2020-02-20-competition-reminder.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+We provide a space for you to work in, with power and internet access, as well as volunteers able to help you with your kits and hands-on guidance with your robots.
```
Comment (PeterJCLaw) [inline SR2020/2020-02-20-competition-reminder.md]: Maybe just drop this? Or replace it with something about what a volunteer should bring (i.e: just themselves).

File: SR2020/2020-02-20-competition-reminder.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+subject: We need competition volunteers!
+---
+
+Hello all,
+
+Quick update on the volunteering opportunities:
+
+## We still need volunteers for the competition!
+
+The dates will be April 18th - 19th, with set-up days on the 16th and 17th if you’re available! The venue is RUSU, the University of Reading Student Union.
```
Comment (RealOrangeOne) [inline SR2020/2020-02-20-competition-reminder.md]: Maybe worth linking to the event page on the website? It's fairly empty now, but will eventually contain more than enough information!

Comment (Adimote) [reply SR2020/2020-02-20-competition-reminder.md]: Done!

**Comment by TomWheal** [comment]
This got sent so it should be merged right?

**Comment by RealOrangeOne** [comment]
It should have done, yes. Good catch @TomWheal !

**Pull request merged**